### PR TITLE
Added the Automated History Analyser, a building unlocked by The phys…

### PR DESCRIPTION
**Pull request by Deepsloth**
…ical Brain.

Discussion: http://www.freeorion.org/forum/viewtopic.php?f=15&t=10157

Post v0.4.6

### CODE REVIEW

**Comment by Vezzra** [comment]
@adrianbroher, not a complaint, just curious: is there a specific reason why you assigned me to that PR...?

**Comment by MatGB** [comment]
Yeah, it's a weird scripting thing that's going to require lots of testing, you've got actually important code to _write_ ;-) (feel free to reassign it to me, not that I'll look at it until we've got time).

**Comment by Vezzra** [comment]
@adrianbroher has actually assigned me to a couple of other issues/PRs where he changed labels, so I guess it is somehow related to his label overhaul, and I will receive proper notice from him what he wants me to do.

**Comment by adrianbroher** [comment]
@Vezzra 

> is there a specific reason why you assigned me to that PR...?

Sorry, no idea. I probably confused this PR with another PR while batch reassigning/relabling/re-"milestoning" issues/PRs.

@MatGB 

> feel free to reassign it to me, not that I'll look at it until we've got time.

I never made it obvious, but assignees is more of a 'this guy know what to do with or how to solve this issue' or 'I want to solve this issue' if self-assigned. By no way you should be obliged to handle that issue now. It's more of a way to see who was in charge of handling the issue in some way (e.g. who merged the issue or who closed/rejected the issue as invalid) when looking back in history.

File: default/scripting/buildings/AUTO_HISTORY_ANALYSER.focs.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        Not Contains Building name = "BLD_AUTO_HISTORY_ANALYSER"
+        OwnedBy empire = Source.Owner
+        TargetPopulation low = 1
+    ]
+    EnqueueLocation = [[ENQUEUE_BUILD_ONE_PER_PLANET]]
+    effectsgroups = [
+        EffectsGroup
+            scope = And [
+                Object id = Source.PlanetID
+                Planet
```
Comment (dbenage-cx) [inline default/scripting/buildings/AUTO_HISTORY_ANALYSER.focs.txt:18]: Is there a need to check `Planet` here after the `Object` match?

File: default/stringtables/en.txt [hunk 2]
```diff
@@ -9478,6 +9478,14 @@ BLD_CULTURE_LIBRARY_DESC
 
 Stores the accumulated knowledge from thousands of years of life on this planet, giving a bonuses to research. This building will be destroyed when the population of the high tech natives reaches zero.'''
 
+BLD_AUTO_HISTORY_ANALYSER
+Automated History Analyser
+
+BLD_AUTO_HISTORY_ANALYSER_DESC
+'''The Automated History Analyser can only be build on a planet with [[buildingtype BLD_CULTURE_ARCHIVES]]. Increases target [[encyclopedia RESEARCH_TITLE]] by 5.
```
Comment (dbenage-cx) [inline default/stringtables/en.txt]: `build` -> `built` (or something similar e.g. constructed)

**Review by dbenage-cx — COMMENTED** [review]
This looks well balanced and a great option.
Minor spelling issue and a question on redundant check, otherwise LGTM.

**Comment by MatGB** [comment]
I'm testing this now and it looks to be a good addition. Obviously, the typo needs fixing, I also have minor AI concerns, using this allows the human player to race even further ahead of the AI in tech and it probably ought to be something many/most AI players build fairly early.

So, @freeorion/ai-team , are you happy to have this merged in and the update the AI when you've time, or would you rather do the update for this and merge this in at the same time? Given it's a minor change I'm inclined for the former but I'd not want this to not be sorted before Release or forgotten about.

**Comment by Morlic-fo** [comment]
If you want to merge this, go ahead but also create a short issue regarding the AI so it is not forgotten.

**Pull request merged**